.eslintrc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     'xwalk/max-cells': [ // DOC: https://github.com/adobe-rnd/eslint-plugin-xwalk/blob/main/docs/rules/max-cells.md
       'error',
       {
-        '*': 4,
+        '*': 5,
```

Comment:
If you're changing the default to 5, then you can remove the exception added below for hero.